src/validator/CreditCardChecking.java

```diff
@@ -0,0 +1,5 @@
+package validator;
+
+public interface CreditCardChecking {
+	public boolean checkValidity(CreditCardVendor ccv, String number);
```

Comment:
Modifier 'public' is redundant for interface methods. You can remove it. 
Also checkValidity name can be simplify to validate.